conan/appveyor/install.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
IMHO, this is too lengthy. I am not sure I get all the abstractions as useful, I would do just:

```python
import os
if os.getenv("APPVEYOR_REPO_TAG") != "true":
    print("Skip...")
else:
    os.system("pip install conan conan-package-tools")
```

Reply:
seems reasonable. I'll adopt your solution.